### DISCUSSION TITLE
I hope to add support for Windows Arm in the Release.

### DISCUSSION THREAD

**Discussion by EmccK** [spacedriveapp/spacedrive#1526] [Feature Request]
I hope to add support for Windows arm64 in the Release.

**Comment by HeavenVolkoff** [comment]
I answered this on discord, so I am just going to copy+paste it here:
<img width="429" alt="image" src="https://github.com/spacedriveapp/spacedrive/assets/9082460/21555c79-526b-4c04-8cf6-1813fa3f7ae5">

**Reply by EmccK** [reply]
Okay, I will pay attention to this issue in the discard.
